test/pages/test_vitals.rb

```
@@ -70,4 +70,74 @@ def test_fn_pmp
     )
     assert_equal('bar', xml.xpath('/r/text()').to_s, xml)
   end
+
+  def test_fn_format_signed
+    {
+      3.3 => ['0.00', '+3.30'],
+      0 => ['0.00', '+0.00'],
+      -1 => ['0.00', '-1.00'],
+      5.123 => ['0.0', '+5.12'],
+      -2.456 => ['0.0', '-2.46'],
+      0.0 => ['0.0', '+0.00'],
```

RuboCop (1.81.7)

[convention] Avoid comma after the last item of a hash.

(Style/TrailingCommaInHashLiteral)